### PR TITLE
Explain how to name concept exercises

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Review by iHiD — APPROVED** [review]
Great! :)

File: anatomy/tracks/concept-exercises.md [hunk 1]
```diff
@@ -450,6 +450,45 @@ Some languages require additional files for the tests to run. Example of these a
...
+
+- `tim-from-marketing`
+- `lucians-luscious-lasagna`
+- `calculator-conundrum`
+
+Bad examples of slugs:
+
+- `TIM-FROM-MARKETING`: should use lowercase (i.e. `tim-from-marketing`)
+- `TimFromMarketing`: should use kebab-case (i.e. `tim-from-marketing`)
+- `floating-point-numbers`: should not use a concept
```
Comment (SaschaMann) [inline anatomy/tracks/concept-exercises.md:488]: *Must* the slugs follow the rules above or *should* they follow those rules?

The paragraph above suggests they must follow those rules, but these examples suggest they're optional.

Comment (iHiD) [reply anatomy/tracks/concept-exercises.md:488]: must.


```suggestion
Bad examples of slugs:

- `TIM-FROM-MARKETING`: does not use lowercase (i.e. `tim-from-marketing`)
- `TimFromMarketing`: does not use kebab-case (i.e. `tim-from-marketing`)
- `floating-point-numbers`: uses a concept name not a story name
```

Comment (SaschaMann) [reply anatomy/tracks/concept-exercises.md:488]: https://github.com/exercism/docs/pull/78

**Pull request merged**